configs/guided_diffusion/README.md

```diff
@@ -47,34 +47,44 @@ We show that diffusion models can achieve image sample quality superior to the c
 You can run adm as follows:
 
 ```python
-from mmagic.utils import register_all_modules
-from mmagic.apis import init_model
+from mmengine import Config, MODELS
+from mmengine.registry import init_default_scope
+from torchvision.utils import save_image
 
-register_all_modules()
-
-# sampling without classifier guidance
-config = 'configs/guided_diffusion/adm_ddim250_8xb32_imagenet-64x64.py'
-ckpt_path = 'https://download.openmmlab.com/mmediting/guided_diffusion/adm-u-cvt-rgb_8xb32_imagenet-64x64-7ff0080b.pth'  # noqa
-model = init_model(config, ckpt_path)
-samples = model.infer(
-            init_image=None,
-            batch_size=4,
-            num_inference_steps=250,
-            labels=None,
-            classifier_scale=0.0,
-            show_progress=True)['samples']
+init_default_scope('mmagic')
 
 # sampling without classifier guidance, CGS=1.0
 config = 'configs/guided_diffusion/adm-g_ddim25_8xb32_imagenet-64x64.py'
```

Comment:
this config has been removed.

Reply:
I still can see this config in main and dev1.x branches, is `configs/guided_diffusion/adm-g_ddim25_8xb32_imagenet-64x64.py` going to be removed?